common/stash/pom.xml

```diff
@@ -31,7 +31,7 @@
         <!-- 3rd-party dependencies -->
         <dependency>
             <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk</artifactId>
+            <artifactId>aws-java-sdk-s3</artifactId>
```

Comment:
Can we also please update the description of the PR saying this is just not an upgrade of AWS SDK, but also replacing with the specific ones - S3, SNS, SQS etc wherever needed instead of the entire sdk?

Reply:
sure